packages/netlify/src/index.ts

```diff
@@ -250,20 +259,9 @@ export default function netlifyIntegration(
 					cacheOnDemandPages: Boolean(integrationConfig?.cacheOnDemandPages),
 					notFoundContent,
 				})});
-				export const config = { name: "Astro SSR", generator: "@astrojs/netlify@${packageVersion}", path: "/*", preferStatic: true };
+				export const config = ${inspect(config)};
```

Comment:
I've used `inspect` and not `JSON.stringify` because we had an issue with accept string literals as keys. This was fixed in https://github.com/netlify/build/pull/5745, but it's not something we can back-port to older versions of the Netlify CLI.

Reply:
Onlythe version is dynamic, so would it be cleaner to inline the whole thing like before?

Reply:
Done in https://github.com/withastro/adapters/pull/315/commits/b8186e7b25507770749fb82f505b41f563a3e946.